course_catalog/api.py

```diff
@@ -391,41 +407,45 @@ def sync_ocw_course(
     if course_json["course_id"] in blacklist:
         is_published = False
 
-    # if course run synced before, update existing Course instance
-    courserun_instance = LearningResourceRun.objects.filter(
-        platform=PlatformType.ocw.value, run_id=uid
-    ).first()
-
-    # Make sure that the data we are syncing is newer than what we already have
-    if (
-        courserun_instance
-        and last_modified <= courserun_instance.last_modified
-        and not force_overwrite
-    ):
-        log.info("Already synced. No changes found for %s", course_prefix)
-        return None
+    log.info("Digesting %s...", course_prefix)
+    course, run = digest_ocw_course(
```

Comment:
This will error if `digest_ocw_course` returns None. Can we add a check if that is the case and return early, similar to this line? https://github.com/mitodl/open-discussions/pull/2567/files?w=1#diff-1458eca3382b327a236a2fd701a27387R387

Reply:
Added a test too